nubis/puppet/files/nubis_users/nubis-users.toml

```diff
@@ -0,0 +1,12 @@
+[template]
+src = "nubis-users.tmpl"
+dest = "/etc/puppet/yaml/nubis_users/common.yaml"
+prefix = "/environment/nubis-users"
```

Comment:
Yes, but that's the model we follow, isolation for each VPC, and given we use terraform to push these type of things anyways, it will be just 3 copies of the same config file, so not much more cumbersome.

And at some point in the future, we might want to be able to roll out and test new configs in stage first, for instance.

It's how we do it for everything else, so I'd like to follow it here too, even if it means a tiny bit of extra complexity (which I doubt) in terms of sticking stuff in Consul.

AFAIK, we'll have 3 Lambda functions in each VPCs, pushing to Consul, so each of them pushing to a different prefix seems like just like different arguments fed via the CloudWatch alarm, right ?